website/docs/Installation/windows-install/installing-podman-with-podman-desktop.md

```diff
@@ -0,0 +1,49 @@
+---
+sidebar_position: 11
+title: Installing Podman on Windows
+description: Podman Desktop can assist you to install Podman on Windows.
+tags: [podman-desktop, podman, installing, windows]
+keywords: [podman desktop, containers, podman, installing, installation, windows]
+---
+
+# Installing Podman on Windows with Podman Desktop
+
+On Windows, running the Podman container engine requires running a Linux distribution on a virtual machine.
+
+Podman Desktop can assist you to install the Podman container engine on the Windows Subsystem for Linux version 2 (WSL 2).
+Main benefits are:
+
+* Ease of use
+* Windows native virtualization performance
+
+#### Prerequisites
+
+* 6 GB RAM
+* No VPN
+* User with administrator privileges
+* Windows 10 Build 18362 or greater, or Windows 11
+* Windows 64bit
+* [Hyper-V enabled](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v)
+* WSL 2 feature enabled, but no Linux virtual machine created. See [Installing WSL 2, step 1 to 5](https://docs.microsoft.com/en-us/windows/wsl/install-manual)
+* Podman is not installed
```

Comment:
it's also possible to use Windows home edition that ahs 'virtualization platform' instead of Hyper-V
https://github.com/containers/podman-desktop/issues/480

Reply:
That explains why it was not working on the vm with the home edition! Haha!